Ginger/GingerCoreCommon/Repository/NewRepositorySerializer.cs

```diff
@@ -587,18 +584,38 @@ private static void SetRepositoryItemHeaderAttr(RepositoryItemHeader RIH, string
         private static void xmlReadListOfObjects(object ParentObj, XmlReader xdr, IObservableList observableList)
         {
             // read list of object into the list, add one by one, like activities, actions etc.
-
             if (ParentObj is RepositoryItemBase && ((RepositoryItemBase)ParentObj).ItemBeenReloaded)
             {
                 observableList.Clear();//clearing existing list items in case it is been reloaded
             }
-
             //TODO: Think/check if we want to make all observe as lazy load
-            if (LazyLoadAttr.Contains(xdr.Name) && observableList.AvoidLazyLoad == false)
+            LazyLoadListConfig lazyLoadConfig = LazyLoadListConfigs.Where(x => x.ListName == xdr.Name).FirstOrDefault();
+            if (ParentObj is RepositoryItemBase && lazyLoadConfig != null && observableList.AvoidLazyLoad == false)
```

Comment:
Codacy found an issue: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5848147)